src/DashboardWidgetExtension/TeamscaleWidget.ts

```diff
@@ -130,16 +130,17 @@ export class TeamscaleWidget {
     }
 
     /**
-     * Loads the Teamscale email contact from the organization settings and assures that an Teamscale server url and project
+     * Loads the Teamscale email contact from the organization settings and assures that a Teamscale server url and project
      * name is set in the Azure DevOps project settings.
      */
     private async loadAndCheckConfiguration() {
         const azureProjectName = VSS.getWebContext().project.name;
         this.projectSettings = new ProjectSettings(Scope.ProjectCollection, azureProjectName);
         this.organizationSettings = new Settings(Scope.ProjectCollection);
-
         this.emailContact = await this.organizationSettings.get(Settings.EMAIL_CONTACT_KEY);
-        return Promise.all([this.initializeTeamscaleClient(), this.initializeNotificationUtils()]);
+
+        await this.initializeNotificationUtils();
+        return Promise.all([this.initializeTeamscaleClient()]);
```

Comment:
question: Why did you pull `initializeNotificationUtils` out of the `Promise.all()` call? 🤔

Reply:
While testing, I ran into some errors where the `initializeTeamscaleClient()` function actually calls the `notificationUtils` while it's still null as it wasn't initialized. So I had to wait for the initialization of the `notificationUtils` first. That was the case for `WidgetConfiguration.ts` and `WorkItemContribution.ts`.

However, for the this file (`TeamscaleWidget.ts`), the `notificationUtils` are already initialized in the constructor, so we don't have to pull it out of the promise call here. I reverted that change now.